Table's cell double click and cell click both fire on single click

### Describe the bug

An added double click event on a table's cell still fires on a single click. 

I believe the issue is the following annotation in `TableCellDoubleClickEvent.java` is the issue
```java
@EventName("dwc-cell-clicked")
```

Maybe this should be double clicked? When trying to add both a single and double click event, an error is thrown saying:

_Caused by: java.lang.IllegalStateException: The event name dwc-cell-clicked is already registered with the event class com.webforj.component.table.event.cell.TableCellClickEvent_


### Reproduction

I added the following two lines to the `TableBasicView.java` file in the controlsamples. Didn't attach all files here as multiple files are required to make the demo run:

```java
    table.onCellClick(e -> App.console().log("Single Click"));
    table.onCellDoubleClick(e -> App.console().log("Double Click"));
```

### Expected behavior

_No response_

### Deployment

WAR (Default)

### System Info

Confirmed this on both a docker instance and using the WAR deployment

--- a/webforj-components/webforj-table/src/main/java/com/webforj/component/table/event/cell/TableCellDoubleClickEvent.java
+++ b/webforj-components/webforj-table/src/main/java/com/webforj/component/table/event/cell/TableCellDoubleClickEvent.java
@@ -12,7 +12,7 @@ import java.util.Map;
  * @author Hyyan Abo Fakher
  * @since 24.00
  */
-@EventName("dwc-cell-clicked")
+@EventName("dwc-cell-dbclicked")
 public class TableCellDoubleClickEvent<T> extends TableCellEvent<T> {
 
   /**

